src/views/awards.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * © 2024 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AwardsNode } from '@manuscripts/transform'
+
+import { createHeader } from '../lib/utils'
+import BlockView from './block_view'
+import { createNodeView } from './creators'
+
+export class AwardsView extends BlockView<AwardsNode> {
+  public elementType = 'div'
+  public wrapper: HTMLElement
+
+  public createElement = () => {
+    this.wrapper = document.createElement(this.elementType)
+    this.wrapper.classList.add('block')
+
+    this.contentDOM = document.createElement(this.elementType)
+    this.contentDOM.className = 'block'
+    this.wrapper.appendChild(this.contentDOM)
+
+    this.dom.appendChild(this.wrapper)
+  }
+
+  public createDOM = () => {
+    this.dom = document.createElement('div')
+    this.dom.classList.add('block-container')
+    this.dom.classList.add(`block-${this.node.type.name}`)
+  }
+
+  public updateContents = () => {
+    if (!this.contentDOM) {
+      return
+    }
+    const header = createHeader(this.node.type.name, 'Funder Information')
+    this.wrapper.insertAdjacentHTML(
```

Comment:
Do we not need to apply any measure here that would prevent duplicated content insertion when an update is triggered on the node?

Reply:
I am pretty sure that for now it is not necessary, because there is no functionality for insert/edit/remove award...

Reply:
we cannot prevent a transaction to run an update on any node. We have plugins that access and modify nodes based on different condition (such as persist or TC) so even if we don't implement any CRUD operations explicitly we still need to make sure an update on an node is safe.

Reply:
OK, thanks for the clarification.